cmd/erasure-server-pool.go

```diff
@@ -936,12 +930,7 @@ func (z *erasureServerPools) DeleteObjects(ctx context.Context, bucket string, o
 		}, j)
 	}
 
-	if err := eg.WaitErr(); err != nil {
-		for i := range derrs {
-			derrs[i] = err
-		}
-		return dobjects, derrs
-	}
```

Comment:
We never had to cancel on error, since we need to provide all the errors appropriately per object to the caller.